pkg/kv/kvserver/scheduler.go

```diff
@@ -26,6 +26,7 @@ import (
 )
 
 const rangeIDChunkSize = 1000
+const maxSchedulerTicks = 10
```

Comment:
Let's use `RaftConfig.RaftElectionTimeoutTicks` for this if we have easy access to it, or otherwise set it to `defaultRaftElectionTimeoutTicks` (15). This would avoid delaying elections.

Reply:
Good idea. Done.